### ISSUE TITLE
Support http/3

### ISSUE THREAD

**Issue by lindenquan** [quarkusio/quarkus#26852]
### Description

[http/3](https://en.wikipedia.org/wiki/HTTP/3) has been supported by Chrome and Edge back in April 2020, and FhirFox in April 2021. Is there any plan to support http/3 ?

**Comment by cescoffier** [comment]
Yes, but we are waiting for Netty 5 to support it. We do not have an ETA (Netty 5 is planned for the end of the year, then we need Vert.x to adopt it and rebase Quarkus on top of this version of Vert.x).

**Comment by mederel** [comment]
Netty 5 is out since January. Now it is about Vert.x.

**Comment by ginccc** [comment]
+1, would love to see this being prioritized 👍

**Comment by cescoffier** [comment]
Unfortunately, it requires netty 5. We don't have a clear ETA about such an integration. But, it's on the roadmap.

**Comment by turing85** [comment]
Still no news?

**Comment by cescoffier** [comment]
No, Netty 5 is delayed, and more or less replaced by Netty 4.2 (planned for this fall). Now, I'm not sure Netty 4.2 will support http/3.

**Comment by indiealexh** [comment]
https://spring.io/blog/2024/11/26/http3-in-reactor-2024

**Comment by cescoffier** [comment]
The HTTP 3 support in Netty is still experimental. I would be careful using it before it becomes stable.

**Comment by Eng-Fouad** [comment]
> The HTTP 3 support in Netty is still experimental. I would be careful using it before it becomes stable.

https://github.com/netty/netty/pull/15048

**Comment by Eng-Fouad** [comment]
Netty 4.2.2.Final was released last week with: [Graduate HTTP/3 from incubator].

**Comment by cescoffier** [comment]
Yes, but the update to Netty 4.2 will wait the bump to Vertx 5.

**Comment by jponge** [comment]
Initial implementation done by @cescoffier
